examples/undocumented/python_modular/neuralnets_simple_modular.py

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env python
-traindat = '../data/fm_train_real.dat'
-testdat = '../data/fm_test_real.dat'
-label_traindat = '../data/label_train_twoclass.dat'
-
-parameter_list = [[traindat,testdat,label_traindat,0.9,1e-3],[traindat,testdat,label_traindat,0.8,1e-2]]
-
-def neuralnets_simple_modular (train_fname, test_fname,
-		label_fname, C, epsilon):
-
-	from modshogun import NeuralLayers, NeuralNetwork, RealFeatures, BinaryLabels
-	from modshogun import Math_init_random, CSVFile
-	Math_init_random(17)
```

Comment:
this example can definitely be translated to meta examples .... nothing that the grammar doesnt support. Just have to flatten the chained method calls